src/test/java/org/broadinstitute/hellbender/utils/HistogramUnitTest.java

```diff
@@ -51,6 +51,15 @@ public void testMedianOfEvens() throws Exception {
         Assert.assertEquals(bimodalHist.median(), 15.0, EPSILON, "");
     }
 
+    @Test
+    public void testMedianEnsureNotHashcodeDependent() throws Exception {
```

Comment:
Add a comment documenting how this test demonstrates that the median calculation is not hashcode dependent, so that we don't lose your hard-won HashMap internals research.

Reply:
done